c2corg_api/views/login.py

```diff
@@ -0,0 +1,43 @@
+from pyramid.view import view_config
```

Comment:
Not sure 'login.py' is the most appropriate name for this file since it contains the logout view as well.
Perhaps auth.py?